singularity/environment.1.0.0.yml

```diff
@@ -1,16 +1,17 @@
-name: snip_test
+name: variantcalling-1.0.0
 channels:
-  - bioconda
   - conda-forge
+  - bioconda
   - defaults
 dependencies:
+  - bbmap=38.18=0
   - bcftools=1.10.2=hd2cd319_0
-  - bedtools=2.29.2=hc088bd4_0
   - bowtie2=2.3.5.1=py37he513fc3_0
   - bwa=0.7.17=hed695b0_7
   - freebayes=1.3.2=py37h7c46602_1
   - htslib=1.10.2=h78d89cc_0
   - minimap2=2.17=h8b12597_1
+  - nextflow=20.04.1=hecc5488_0
```

Comment:
You shouldn't need to install Nextflow in the Conda environment.

Reply:
Nextflow has been removed from conda.